Makefile

```
@@ -10,23 +10,29 @@ include mk/config.mk
 include mk/schemes.mk
 include mk/rules.mk
 
+Q := @
```

I guess this should be 
```suggestion
Q ?= @
```